### PR TITLE
Add view console as a seperate permission in subusers

### LINKED ISSUE #3466: Add view console as a seperate permission in subusers.
**Is your feature request related to a problem? Please describe.**
Well, I have a moderator team. I want them to have the ability to start the server if it crashes while I'm asleep. However, if I give them the start power action, they will have permissions to view the console. That contains sensitive info, although they can't type commands, they can still collect IP addresses and it's just not a good idea to show mods this

**Describe the solution you'd like**
I'd like to have a new permission called "View Console" that can give people permission to view it or not. Giving people the start/stop/kill permissions will only give them access to the buttons, but to now view the console itself, unless they have the view console permission

**Describe alternatives you've considered**
Only possible alternative I have considered is trying to use discord bot along with pterodactyl API to make a simple .start command for the mods to do this. But it'd be neat and better to do it on the panel itself!

**Additional context**

### PR DESCRIPTION
**Pull request by Jelcoo**
Closes #3466

### CODE REVIEW

File: app/Models/Permission.php [hunk 1]
```diff
@@ -16,7 +16,8 @@ class Permission extends Model
      * Constants defining different permissions available.
      */
     public const ACTION_WEBSOCKET_CONNECT = 'websocket.connect';
-    public const ACTION_CONTROL_CONSOLE = 'control.console';
+    public const ACTION_CONTROL_CONSOLE_SEND = 'control.sendconsole';
+    public const ACTION_CONTROL_CONSOLE_READ = 'control.sendconsole';
```
Comment (Boy132) [inline app/Models/Permission.php:20]: That should be `control.readconsole`. :)

File: app/Models/Permission.php [hunk 2]
```diff
@@ -16,7 +16,8 @@ class Permission extends Model
      * Constants defining different permissions available.
      */
     public const ACTION_WEBSOCKET_CONNECT = 'websocket.connect';
-    public const ACTION_CONTROL_CONSOLE = 'control.console';
+    public const ACTION_CONTROL_CONSOLE_SEND = 'control.sendconsole';
```
Comment (matthewpi) [inline app/Models/Permission.php]: Use dashes when multiple words are being used rather than putting them together.  `control.send-console` for example.

**Review by matthewpi — CHANGES_REQUESTED** [review]

File: app/Models/Permission.php [hunk 3]
```diff
@@ -120,7 +121,8 @@ class Permission extends Model
         'control' => [
             'description' => 'Permissions that control a user\'s ability to control the power state of a server, or send commands.',
             'keys' => [
-                'console' => 'Allows a user to send commands to the server instance via the console.',
+                'sendconsole' => 'Allows a user to send commands to the server instance via the console.',
+                'readconsole' => 'Allows a user to read the console.',
```
Comment (matthewpi) [inline app/Models/Permission.php]: You missed adding the `-` in these

**Review by matthewpi — CHANGES_REQUESTED** [review]

**Review by matthewpi — CHANGES_REQUESTED** [review]
Marking this as requesting changes so it doesn't get merged.  Adding/changing these permissions also requires changing them on Wings.

**Comment by DaneEveritt** [comment]
Closing as this is both a breaking permissions change and requires updates to Wings that are not present.

https://github.com/pterodactyl/panel/blob/develop/CONTRIBUTING.md

**Pull request closed without merging**